lib/workers/repository/reconfigure/utils.ts

```diff
@@ -1,3 +1,70 @@
+import is from '@sindresorhus/is';
+import JSON5 from 'json5';
+import type { RenovateConfig } from '../../../config/types';
+import { logger } from '../../../logger';
+import { platform } from '../../../modules/platform';
+import { scm } from '../../../modules/platform/scm';
+import type { BranchStatus } from '../../../types/branch-status';
+import { readLocalFile } from '../../../util/fs';
+import { detectConfigFile } from '../init/merge';
+
 export function getReconfigureBranchName(prefix: string): string {
   return `${prefix}reconfigure`;
 }
+
+export async function setBranchStatus(
+  branchName: string,
+  description: string,
+  state: BranchStatus,
+  context?: string | null,
+): Promise<void> {
+  if (!is.nonEmptyString(context)) {
+    // already logged this case when validating the status check
+    return;
+  }
+
+  await platform.setBranchStatus({
+    branchName,
+    context,
+    description,
+    state,
+  });
+}
+
+export async function getReconfigureConfig(branchName: string): Promise<{
+  config: RenovateConfig | null;
+  errMessage?: string;
+  configFileName?: string;
+}> {
+  let errMessage = undefined;
+
+  await scm.checkoutBranch(branchName);
+  const configFileName = await detectConfigFile();
+
+  if (is.null_(configFileName)) {
```

Comment:
```suggestion
  if (configFileName === null) {
```

`is.null_` isn't an improvement and will be renamed to `is.null` in next is major. so don't use it to have less migrations to do. 🙃